packages/alfa-style/src/property/z-index.ts

```diff
@@ -0,0 +1,16 @@
+import { Integer, Keyword } from "@siteimprove/alfa-css";
+import { Parser } from "@siteimprove/alfa-parser";
+
+import { Longhand } from "../longhand.js";
+
+const { either } = Parser;
+
+const parse = either(Keyword.parse("auto"), Integer.parse);
+
+/**
+ * {@link https://developer.mozilla.org/en-US/docs/Web/CSS/z-index}
+ * @internal
+ */
+export default Longhand.of(Keyword.of("auto"), parse, (value) => value, {
```

Comment:
**question:** Do we need to resolve calculated values (for integers)? 
I think it is not technically needed (since `z-index: calc(1em / 1px)` sounds like madness), but we should have the computed type as `Integer.Canonical` which may require a call to `.resolve` anyway (using the default length resolver based on the style, same as in other properties).